src/main/java/de/dm/infrastructure/logcapture/LogEventMatcher.java

```diff
@@ -27,14 +27,14 @@
      *
      * @return matched aspect
      */
-    String getMatcherDescription();
+    String getMatcherTypeDescription();
 
     /**
-     * returns an error message describing why a logged event does match, even of it shouldn't.
+     * returns an error message describing the concrete matcher, including as much of its expectations as possible
      *
      * @return matched aspect
      */
-    String getMatchingErrorMessage();
+    String getMatcherDetailDescription();
```

Comment:
In short: Because it was used ambiguously and not only for the error case. Also, the description is easier to formulate if it only needs to describe the matcher and does not need to concern itself with the context in which it is used.

Let's discuss this in detail in person.